src/main/cpp/t04_count.cpp

```diff
@@ -28,5 +28,15 @@
 using namespace std;
 
 int t04_count() {
+	int cou = 1, pos = 0;
+    string s;
+    getline(cin, s);
 
+    while((int)s.find(" ", pos) > -1){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Inefficient usage of string::find() in condition; string::starts_with() could be faster.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513044)